docs/src/index.md

```diff
@@ -41,7 +41,17 @@ xtargetcols = DA(repeat(xtarget, 1, nlon, nlat))
 fsourcecols = DA(sin.(xsourcecols))
 ftargetcols = DA(zeros(FT, ntarget, nlon, nlat))
 
+# interpolate with different source grid and target grids for all columns
```

Comment:
Could you emphasize in a comment what the difference is between `xsourcecols` and `xsource` (and therefore between `xtargetcols` and `xtarget`) please?